src/main/java/net/sf/jabref/logic/util/io/SimpleFileList.java

```diff
@@ -0,0 +1,117 @@
+/*  Copyright (C) 2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.logic.util.io;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.util.io.FileUtil;
+import net.sf.jabref.logic.util.io.SimpleFileListEntry;
+import net.sf.jabref.logic.util.strings.StringUtil;
+
+/**
+ * Data structure to contain a list of file links, parseable from a coded string.
+ * To be used instead of FileListTableModel when no GUI parts are required.
+ */
+public class SimpleFileList {
```

Comment:
I wouldn't use a new class to wrap `List<SimpleFileListEntry>`. All the decode methods should be moved to the `FileUtil` class.